content/en/opentelemetry/guide/ingestion_sampling_with_opentelemetry.md

```diff
@@ -63,7 +63,9 @@ The [Tail Sampling Processor][9] and [Probabilistic Sampling Processor][10] allo
 
 **Note**: Tail sampling's main limitation is that all spans for a given trace must be received by the same collector instance for effective sampling decisions. If the trace is distributed across multiple collector instances, there’s a risk that some parts of a trace are dropped whereas some other parts of the same trace are sent to Datadog.
 
-With collector-level tail-based sampling, the APM metrics are computed on the sampled traffic that the Datadog Exporter sees. Use the [ingestion volume control guide][11] to read more about the implications of setting up trace sampling on trace analytics monitors and metrics from spans.
+To ensure that APM metrics are computed based on 100% of the applications' traffic while using collector-level tail-based sampling, preprend the [Datadog Processor][11] in front of your sampling processor in the collectors' traces pipeline. The processor is available starting from the v0.69.0 of the OpenTelemetry Collector Contrib release.
```

Comment:
```suggestion
To ensure that APM metrics are computed based on 100% of the applications' traffic while using collector-level tail-based sampling, preprend the [Datadog Processor][11] in front of your sampling processor in the collectors' traces pipeline. The processor is available with OpenTelemetry Collector Contrib v0.69.0+.
```

Reply:
Thanks! I think we were both having trouble with this one :))